### PR TITLE
Created codeowners file

### PR DESCRIPTION
**Pull request by RachalCassity**
Created CODEOWNERS file.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+#  @department-of-veterans-affairs/backend-review-group will be requested for
+# review when someone opens a pull request.
+
+*     @department-of-veterans-affairs/backend-review-group
```
[the remark this answers is not in the document]
Comment (RachalCassity) [reply .github/CODEOWNERS]: @LindseySaari Yeah, it's [platform-console-services](https://github.com/orgs/department-of-veterans-affairs/teams/platform-console-services)

File: catalog-info.yaml [hunk 2]
```diff
@@ -5,15 +5,36 @@ metadata:
...
   annotations:
     github.com/project-slug: department-of-veterans-affairs/platform-console-api
+    backstage.io/techdocs-ref: url:https://github.com/department-of-veterans-affairs/platform-console-api
   tags:
     - ruby
     - rails
     - platform-console-api
+  links:
+    - url: http://platform-console-api.vfs.va.gov
+      title: Platform Console API - Utility
```
Comment (LindseySaari) [inline catalog-info.yaml:15]: Did you add these items or is this a merge conflict?

Comment (RachalCassity) [reply catalog-info.yaml]: This was removed.

**Review by LindseySaari — APPROVED** [review]

**Comment by LindseySaari** [comment]
@RachalCassity 👍 Merge away!

**Pull request merged**